sources/platform/actors/publishing/badge.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Actor Status Badge
+description: Apify has a QA system that regularly runs automated tests to ensure that all Actors in the store are functional.
+slug: /actors/publishing/status-badge
+sidebar_position: 4
+---
+
+Apify Actor Status Badge can be embedded in 3rd party documentation to show current health status and engagement of the Actor in the Apify Platform.
+
+---
+
+### Actor Status Badge Example
+
+This is the badge generated for the [Apify's Website Content Crawler](https://apify.com/apify/website-content-crawler Actor).
+
+[![Website Content Crawler Actor](https://apify.com/actor-badge?actor=apify/website-content-crawler)](https://apify.com/apify/website-content-crawler)
+
+
+### How to embed the Actor Status Badge
+
+The Badge is a dynamic SVG image loaded from the Apify platform. The Badge is served from the URL Template:
+
+```text
+https://apify.com/actor-badge?actor=:username/:actor-name
+```
+
+In order to embed the badge in the HTML documentation, just use it as an image wrapped in a link as shown in the example below. Don't froget to use the `username` and `actor-name` of your Actor.
+
+#### HTML Example
+
+```html
+<a href="https://apify.com/apify/website-content-crawler">
+  <img src="https://apify.com/actor-badge?actor=apify/website-content-crawler">
+</a>
+```
+
+#### Markdown Example
+
+```markdown
+[![Website Content Crawler Actor](https://apify.com/actor-badge?actor=apify/website-content-crawler\)](https://apify.com/apify/website-content-crawler)
+```
+
+### Supported Actor States
+
+The badge indicates the state of the Actor in the Apify Platform as the result of the [automated testing](../development/automated_tests.md).
+
+#### Actor OK
+
+![Actor badge OK](./images/actor-badge/ok.svg)
+
+#### Actor Under Maintenance
+
+![Actor badge under maintenance](./images/actor-badge/under-maintenance.svg)
+
+#### Actor Deprecated
+
+![Actor badge deprecated](./images/actor-badge/deprecated.svg)
+
+#### Actor Not Found
+
+![Actor badge not found](./images/actor-badge/not-found.svg)
+
+### Example of the Badge in Github Readme
+
+![Actor badge in githhub readme](images/github-badge-screenshot.png)
```

Comment:
While having the picture alone here is great, could we maybe attach a code example that results in this picture? so the user has one place where he can both see the code examples & see the expected results?

Reply:
Ideally we would have the same markdown that leads to the results seen in the image the we are providing.